network/server_test.go

```diff
@@ -491,3 +499,43 @@ func TestSelfConnection_WithBootNodes(t *testing.T) {
 		})
 	}
 }
+
+func TestMinimumBootNodeCount(t *testing.T) {
+	tests := []struct {
+		name      string
+		bootNodes []string
+		isError   bool
+	}{
+		{
+			name:      "Server config with empty bootnodes",
+			bootNodes: []string{},
+			isError:   true,
+		},
+		{
+			name:      "Server config with less than two bootnodes",
+			bootNodes: []string{"/ip4/127.0.0.1/tcp/10001/p2p/16Uiu2HAmJxxH1tScDX2rLGSU9exnuvZKNM9SoK3v315azp68DLPW"},
```

Comment:
Please consider not using hard-coded values for these peerIDs, instead generate them during the test.

There's no reason to hard-code them

Reply:
I felt this generation is unnecessary overhead, since this test scope is to verify whether the bootnode count constraints are satisfied or not.

Reply:
I get that we're just trying to check if the size of the bootnodes array is correct.
It's just misleading to have hardcoded values - it gives the impression that these IDs are somehow relevant to the test.

The overhead of generating new keys is minimal to non-existent

Reply:
That makes sense, Here's the commit [link](https://github.com/0xPolygon/polygon-sdk/pull/260/commits/2db9caaecbc7e5e9186e16750333423a91670c54)